Assembly-CSharp/Global/UI/UIKey/UIKeyTrigger.cs

```diff
@@ -783,6 +795,25 @@ public Boolean ContainsAndroidQuitKey()
         return false;
     }
 
+    private Boolean shouldTurboDialog(List<Control> confirmKeys)
+    {
+        if (!Configuration.Cheats.TurboDialog || preventTurboKey || TimerUI.Enable || PreventTurboOnFields())
+            return false;
+
+        return TurboKey || ((HonoInputManager.Instance.IsInput(Control.RightBumper) || ShiftKey) && confirmKeys.Any(HonoInputManager.Instance.IsInput));
+    }
+
+    public Boolean PreventTurboOnFields() // [DV] TODO: Make it compatible with DictionaryPatch
+    {
+        List<Int32> fieldidpreventturbo = new List<Int32> { 656, 657, 658, 659, 2950, 2951, 2952 }; // Kwe Marsh + Chocobo Minigame places
+        foreach (Int32 id in fieldidpreventturbo)
+        {
+            if (FF9StateSystem.Common.FF9.fldMapNo == id)
+                return true;
+        }
+        return false;
+    }
```

Comment:
Can be replaced by:
```csharp
// Kwe Marsh + Chocobo Minigame places
public static HashSet<Int32> FieldsWithDialogTurboDisabled = [ 656, 657, 658, 659, 2950, 2951, 2952 ];
public static Boolean FieldPreventDialogTurbo => FieldsWithDialogTurboDisabled.Contains(FF9StateSystem.Common.FF9.fldMapNo);
```
Having the HashSet public will make it easier to be modifiable with ``DictionaryPatch.txt`` later